v2/cmd/asoctl/cmd/import_azure_resource.go

```diff
@@ -47,6 +47,25 @@ func newImportAzureResourceCommand() *cobra.Command {
 
 	cmd.MarkFlagsMutuallyExclusive("output", "output-folder")
 
+	cmd.Flags().StringVarP(
+		&options.namespace,
+		"namespace",
+		"n",
+		"",
+		"Write the imported resources to the specified namespace")
+	cmd.Flags().StringSliceVarP(
+		&options.labels,
+		"label",
+		"l",
+		nil,
+		"Add the specified labels to the imported resources. Multiple comma-separated labels can be specified (example.com/mylabel=foo,example.com/mylabel2=bar) or the --label (-l) argument can be used multiple times (-l example.com/mylabel=foo -l example.com/mylabel2=bar)")
```

Comment:
Can you put a space after the comma, e.g.

```
--label example.com/mylabel=foo, example.com/mylabel2=bar
```

Also, the examples need to be consistent:

```suggestion
		"Add the specified labels to the imported resources. Multiple comma-separated labels can be specified (--label example.com/mylabel=foo,example.com/mylabel2=bar) or the --label (-l) argument can be used multiple times (-l example.com/mylabel=foo -l example.com/mylabel2=bar)")
```